.drone.yml

```diff
@@ -42,6 +42,19 @@ services:
       - NODE_MAX_SESSION=5
 
 pipeline:
+  composer-change-patch:
+    group: warmup
+    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
+    volumes:
+      - /cache:/cache
+    commands:
+      # Change the patch for 10.2.x.
```

Comment:
Let's do the opposite, add 10.2.x patch in composer.json and replace it for 10.1. We install by default on 10.2 from the composer.json of the component.